Tipstaff.Infrastructure/Repositories/RespondentRepository.cs

```diff
@@ -78,7 +78,7 @@ public Respondent GetRespondentByKeys(string id, string rangeKey)
 
         public void Update(Respondent respondent)
         {
-            var entity = _dynamoAPI.GetEntityByKeys(respondent.Id, respondent.TipstaffRecordID);
+            var entity = _dynamoAPI.GetEntityByKeys(respondent.Id, respondent.Id);
```

Comment:
I don't fully understand this - if respondents are not a dynamo table anymore, how does the update work? _dynamoAPI is created to point to Respondent dynamo table, but that has been removed from the DynamoTables/Respondent.cs